webpack/assets/javascripts/compute_resource/ovirt.js

```diff
@@ -121,9 +122,12 @@ function addVolume({
 
   disableElement($(`[id$=${newId}_size_gb]`).val(sizeGb));
   disableElement($(`[id$=${newId}_storage_domain]`).val(storageDomain));
-  disableElement($(`[id$=${newId}_wipe_after_delete]`).val(wipeAfterDelete));
+  disableElement($(`[id$=${newId}_preallocate]`).prop('checked', !sparse));
   disableElement($(`[id$=${newId}_interface]`).val(diskInterface));
   disableElement($(`[id$=${newId}_bootable_true]`).attr('checked', bootable));
+  disableElement(
+    $(`[id$=${newId}_wipe_after_delete]`).prop('checked', wipeAfterDelete)
```

Comment:
For some reason this is not disabled on edit. This is even before your PR but maybe if you are already here this can be fixed.